docs/en/get_started/quickstart.md

```diff
@@ -1,3 +1,275 @@
+```
+
+Explanation of the config name:
+
+| Config Name | internlm2_7b_qlora_colorist_e5 |
+| ----------- | ------------------------------ |
+| Model Name  | internlm2_7b                   |
+| Algorithm   | qlora                          |
+| Dataset     | colorist                       |
+| Epochs      | 5                              |
+
+The directory structure at this point should look like this:
+
+```bash
+.
+├── colors
+│   ├── colors.json
+│   ├── dataset_infos.json
+│   ├── README.md
+│   └── train.jsonl
+├── internlm2_7b_qlora_colorist_e5_copy.py
+└── Shanghai_AI_Laboratory
+    └── internlm2-chat-7b
+        ├── config.json
+        ├── configuration_internlm2.py
+        ├── configuration.json
+        ├── generation_config.json
+        ├── modeling_internlm2.py
+        ├── pytorch_model-00001-of-00008.bin
+        ├── pytorch_model-00002-of-00008.bin
+        ├── pytorch_model-00003-of-00008.bin
+        ├── pytorch_model-00004-of-00008.bin
+        ├── pytorch_model-00005-of-00008.bin
+        ├── pytorch_model-00006-of-00008.bin
+        ├── pytorch_model-00007-of-00008.bin
+        ├── pytorch_model-00008-of-00008.bin
+        ├── pytorch_model.bin.index.json
+        ├── README.md
+        ├── special_tokens_map.json
+        ├── tokenization_internlm2_fast.py
+        ├── tokenization_internlm2.py
+        ├── tokenizer_config.json
+        └── tokenizer.model
+```
+
+## Modify the config
+
+In this step, we need to modify the model path and dataset path to local paths and modify the dataset loading method.
+In addition, since the copied config is based on the Base model, we also need to modify the `prompt_template` to adapt to the Chat model.
+
+```python
+#######################################################################
+#                          PART 1  Settings                           #
+#######################################################################
+# Model
+# pretrained_model_name_or_path = 'internlm/internlm2-7b'
+pretrained_model_name_or_path = './Shanghai_AI_Laboratory/internlm2-chat-7b'
+
+# Data
+# data_path = 'burkelibbey/colors'
+data_path = './colors/train.jsonl'
+# prompt_template = PROMPT_TEMPLATE.default
+prompt_template = PROMPT_TEMPLATE.internlm2_chat
+
+...
+#######################################################################
+#                      PART 3  Dataset & Dataloader                   #
+#######################################################################
+train_dataset = dict(
+    type=process_hf_dataset,
+    # dataset=dict(type=load_dataset, path=data_path),
+    dataset=dict(type=load_dataset, path='json', data_files=dict(train=data_path)),
+    tokenizer=tokenizer,
+    max_length=max_length,
+    dataset_map_fn=colors_map_fn,
+    template_map_fn=dict(
+        type=template_map_fn_factory, template=prompt_template),
+    remove_unused_columns=True,
+    shuffle_before_pack=True,
+    pack_to_max_length=pack_to_max_length)
+```
```

Comment:
Please use `diff` markdown block (like [here](https://github.com/InternLM/xtuner/tree/main/examples/demo_data/multi_turn_2#config)), instead of `python + comment`